utils/progressbar/progressbarmng.go

```diff
@@ -194,14 +205,53 @@ func (bm *ProgressBarMng) NewStringProgressBar(headline string, updateFn func()
 		mpb.BarRemoveOnComplete(),
 		mpb.PrependDecorators(
 			decor.Name(headline),
-			decor.Any(func(statistics decor.Statistics) string {
-				return updateFn()
-			}),
 		),
 	)
 	return pb
 }
 
+// A progress bar with two counters values shown on the right side of the progress bar, the first value controls what the bar shows
+// The total tasks1 can be passes by an int or *int, if you want to use it with int send nil to the optional total and done tasks1 and the wanted totalTasks to total tasks1
+// else if you want to use
+func (bm *ProgressBarMng) NewDoubleValueProgressBar(headLine1 string, headLine2 string, color Color, totalTasks1 int64, OptionalTotalTasks1, optionalDoneTasks1, totalTasks2, doneTaks2 *int64) *TasksProgressBar {
```

Comment:
Let's rename and reorder thre arguments list (for example, it makes more sense to have the color as the last argument).